corehq/apps/reports/standard/cases/basic.py

```diff
@@ -24,6 +26,129 @@
 from .data_sources import CaseInfo, CaseDisplay
 
 
+class CaseExportScope(object):
```

Comment:
This is an awful lot of copy-pasted code - this looks like it would be confusing to update without context.  I'm guessing this is pulled out so that the descendants of `CaseListMixin` don't inherit this behavior automatically.  If that's the case, I don't think that's something we need to worry about.

The behavior only changes for restricted users, right?  In that case it won't matter for regular users whether this code is here or not.  Restricted users won't be able to access descendants of `CaseListMixin` anyways unless those reports are marked as location-safe, in which case they'd want this behavior.

Would it be possible to just include all this in the `CaseListMixin`?